pkg/extensions/extension_sync.go

```diff
@@ -19,6 +23,13 @@ func EnableSyncExtension(config *config.Config, repoDB repodb.RepoDB,
 		onDemand := sync.NewOnDemand(log)
 
 		for _, registryConfig := range config.Extensions.Sync.Registries {
+			// in order to help configure sync in cluster mode, ignore URLs pointing to this server
+			selfAddress := net.JoinHostPort(config.HTTP.Address, config.HTTP.Port)
```

Comment:
What if zot is configured to listen in 0.0.0.0? Or in one place of the config it is listed with the hostname and in the other with the IP?

Reply:
yeah, I'm not sure how to solve this...